.changes/next-release/bugfix-AWSS3-3112634.json

```diff
@@ -0,0 +1,5 @@
+{
+    "category": "AWS S3", 
+    "type": "bugfix", 
+    "description": "Skip Etag validation for KMS encrypted objects"
```

Comment:
minor, can we update the description to something like "Use request header to determine if  checksum validation should be enabled for `s3#putObject`"